announcements/_posts/2019-10-25-w09-pi-fest.md

```diff
@@ -0,0 +1,36 @@
+---
+author: Jeffery Russell (@jrtechs)
+title: "Pi Fest"
+layout: post-event
+date-start: "2019-10-25 16:30"
+date-end: "2019-10-25 18:00"
+location: "GOL-2620 (Med. DB Lab)"
+---
+
+Hey Ritluggers,
+
+This week we will be exploring all the wonderful things you can do with Raspberry Pis.
+We will go over some Pi 101 things and then dive into demoing some of our favorite Raspberry pi projects.
+If anyone in the club has a Raspberry Pi, feel free to bring it with you to the club this Friday.
+
+Tomorrow (Thursday) there will be a "pizza social" during FOSS hours where we are meeting and greeting FOSS students faculty and alums.
```

Comment:
"...where there will be a meet-and-greet with other FOSS students, faculty, and alums."